packages/input_schema/src/schema.json

```diff
@@ -32,22 +32,80 @@
             "type": "object",
             "patternProperties": {
                 "^": {
-                    "oneOf": [
-                        { "$ref": "#/definitions/stringProperty" },
-                        { "$ref": "#/definitions/stringEnumProperty" },
-                        { "$ref": "#/definitions/arrayProperty" },
-                        { "$ref": "#/definitions/objectProperty" },
-                        { "$ref": "#/definitions/integerProperty" },
-                        { "$ref": "#/definitions/booleanProperty" },
-                        { "$ref": "#/definitions/anyProperty" }
-                    ]
+                    "if": {
+                        "properties": {
+                            "resourceType": {
+                                "not": {
```

Comment:
Nit: why not in condition? I prefer condition without not and switch else branches, it will make it more readable.